vehicle/pacmod_interface/README.md

```diff
@@ -1,15 +1,80 @@
-# How to build and use AS node in Autoware
+# pacmod_interface
 
-1. Clone the autoware repository
-2. After cloning autoware get all the submodules.
+`pacmod_interface` is the package to connect Autoware with Pacmod.
 
-   ```sh
-   git submodule update --init --recursive
-   ```
+## Input / Output
 
-3. Build the workspace
-4. There is no GUI interface to launch this node. Hence use,
+### Input topics
 
-   ```sh
-   roslaunch as ssc_interface.launch
-   ```
+- From Autoware
+
+  | Name                                   | Type                                                     | Description                                           |
+  | -------------------------------------- | -------------------------------------------------------- | ----------------------------------------------------- |
+  | `/control/command/control_cmd`         | autoware_auto_control_msgs::msg::AckermannControlCommand | lateral and longitudinal control command              |
+  | `/control/command/gear_cmd`            | autoware_auto_vehicle_msgs::msg::GearCommand             | gear command                                          |
+  | `/control/command/turn_indicators_cmd` | autoware_auto_vehicle_msgs::msg::TurnIndicatorsCommand   | turn indicators command                               |
+  | `/control/command/hazard_lights_cmd`   | autoware_auto_vehicle_msgs::msg::HazardLightsCommand     | hazard lights command                                 |
+  | `/vehicle/engage`                      | autoware_auto_vehicle_msgs::msg::Engage                  | engage command                                        |
+  | `/vehicle/command/actuation_cmd`       | autoware_vehicle_msgs::msg::ActuationCommandStamped      | actuation (accel/brake pedal, steering wheel) command |
+  | `/control/command/emergency_cmd`       | autoware_vehicle_msgs::msg::VehicleEmergencyStamped      | emergency command                                     |
+
+- From Pacmod
+
+  | Name                                | Type                              | Description                                                             |
+  | ----------------------------------- | --------------------------------- | ----------------------------------------------------------------------- |
+  | `/pacmod/parsed_tx/steer_rpt`       | pacmod3_msgs::msg::SystemRptFloat | current steering wheel angle                                            |
```

Comment:
due to current change pacmod3 parse tx and parse rx is removed and it is devided by hoge_rpt and hoge_cmd

Reply:
@taikitanaka3 could you tell me which PR the change is included?

Reply:
@taikitanaka3 how about this one https://github.com/tier4/AutowareArchitectureProposal.iv/pull/616/commits/d96d76dd1595615be4c5307c947f247e50720255